### PR TITLE
Update Plexus.csv

### PR DESCRIPTION
**Pull request by UnixPhonez**
Added 10 apps (iAttended, Navy Federal Credit Union, SchoolFirst FCU, GroupMe, Canon Camera Connect, Outlook, PayRange, MyCU, SimpleLogin, Govee Home, BLEDIM

### CODE REVIEW

File: src/data/Plexus.csv [hunk 1]
```diff
@@ -1022,3 +1022,19 @@ Zombie Tsunami,January,2021,4,No reported issues,X,X
 Zoom,December,2020,4,No reported issues,4,No reported issues
 Zulip,September,2020,3,Notifications not working,X,X
 Zwift Companion,March,2021,X,X,4,No reported issues
+iAttended,January,2022,2,App launches with no warnings but when you go to login via email link, doesn't log you in,X,X
```
Comment (Nairou) [inline src/data/Plexus.csv:1025]: Comment contains a comma, needs to be quoted

File: src/data/Plexus.csv [hunk 2]
```diff
@@ -1022,3 +1022,19 @@ Zombie Tsunami,January,2021,4,No reported issues,X,X
...
+Navy Federal Credit Union,January,2022,4,No reported issues,X,X
+SchoolFirst FCU,January,2022,4,No reported issues,X,X
+GroupMe,January,2022,1,Doesn't let you use without google services,X,X
+Canon Camera Connect,January,2022,4,No reported issues,X,X
+Outlook,January,2022,3,No notifications,X,X
+PayRange,January,2022,3,Says can't work without Google services, although still lets you log in and scan for washer machines,X,X
```
Comment (Nairou) [inline src/data/Plexus.csv:1031]: Comment contains a comma, needs to be quoted

**Review by Nairou — CHANGES_REQUESTED** [review]

File: src/data/Plexus.csv (hunk 1 above)
Comment (UnixPhonez) [reply src/data/Plexus.csv]: fixed

File: src/data/Plexus.csv (hunk 2 above)
Comment (UnixPhonez) [reply src/data/Plexus.csv]: fixed

**Comment by henry-fisher** [comment]
Hey there! Thank you for the patience, we've been hard at work on the Plexus app. Now that it's live, we've migrated to that system of submitting apps conveniently from your mobile device. I'll be closing this PR so that people can migrate to submitting apps directly from the mobile app. Thank you so much for your contribution and hope to see you there! 
Download the app here: https://github.com/techlore/Plexus-app?tab=readme-ov-file#download

**Pull request closed without merging**